test/cmd/get.sh

```diff
@@ -23,7 +23,7 @@ os::cmd::expect_success "_output/oshinko-cli scale abc --workers=2 --token=`oc w
 os::cmd::try_until_text "_output/oshinko-cli get abc --token=`oc whoami -t` -o json" '"workerCount": 0' 2
 #delete
 os::cmd::expect_success "_output/oshinko-cli delete abc --token=`oc whoami -t`"
-VERBOSE=true os::cmd::expect_success "_output/oshinko-cli get abc --token=`oc whoami -t` -o json"
+VERBOSE=true os::cmd::expect_failure_and_text "_output/oshinko-cli get abc --token=`oc whoami -t` -o json" "no such cluster 'abc'"
```

Comment:
Note, this change is duplicated from https://github.com/radanalyticsio/oshinko-cli/pull/36